pkg/strings/strings.go

```diff
@@ -1,12 +1,12 @@
-package common
+package strings
 
 import "github.com/telepresenceio/telepresence/rpc/v2/manager"
 
-func ModeToString(mode *manager.Mode) string {
+func FromMode[M ~int32](mode M) string {
```

Comment:
Using generics here seem like overkill to me. What's wrong with using `manager.Mode` (not a pointer)?